MarioAI-Base/src/main/java/ch/idsia/Constants.java

```diff
@@ -0,0 +1,5 @@
+package ch.idsia;
+
+public class Constants {
```

Comment:
Perhaps ApplicationConstants would be a more precise name. I wouldn't want people adding a constant like PI to this class.